DebuggableASTInterpreter/DASTContext.class.st

```diff
@@ -68,8 +68,17 @@ DASTContext class >> newWithSender: aDASTContext receiver: aReceiver messageNode
 
 { #category : #'instance creation' }
 DASTContext class >> valueToBlockMessages [
-	^ valueToBlockMessages 
-		ifNil: [ valueToBlockMessages := (BlockClosure methodDict select: [ :e | (201 to: 206) includes: e primitive ]) collect: [ :e | e selector ] ]
+
+	^ valueToBlockMessages ifNil: [ 
+		  valueToBlockMessages := (BlockClosure methodDict select: [ :e | "Is it really better than `(205 to: 209) includes: e primitive` ?"
+			                           #( #value #value: #value:value:
+			                              #value:value:value:
+			                              #value:value:value:value:
+			                              #value #value:value: #value:value:
+			                              #valueNoContextSwitch
+			                              valueNoContextSwitch:
+			                              #valueWithArguments ) includes:
+				                           e selector ]) keys ]
```

Comment:
Why do you need to select, then keys? why not just returning the list of selectors? :)

Reply:
Indeed, I'm dumb :) . I was to focused on the original expression